src/main/frontend/components/bug_report.cljs

```diff
@@ -44,8 +44,9 @@
                                     (util/copy-to-clipboard! result)
                                     (notification/show! "Copied to clipboard!"))
 
-        reset-step! (fn [] ((set-step! 0)
-                            (set-result! {})))]
```

Comment:
Haven't noticed that there is an extra `()`, sorry.

Reply:
Never mind. I noticed some error logs while clicking around buttons.